.github/config.yml

```diff
@@ -3,19 +3,23 @@
 # Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
 
 # Comment to be posted to on first time issues
-newIssueWelcomeComment: >
+newIssueWelcomeComment: |
   Thanks for opening your first issue here! Please follow the issue template to help us help you 👍🎉😄
-
+  If you have screenshots to share demonstrating the issue, that's really helpful! 📸 You can [make a gif](https://www.cockos.com/licecap/) too!
 # Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
 
 # Comment to be posted to on PRs from first time contributors in your repository
-newPRWelcomeComment: >
-  Thanks for opening this pull request! `Dangerbot` will test out your code and reply in a bit with some pointers and requests. There may be some errors, but don't worry! We're here to help! 👍🎉😄
-
+newPRWelcomeComment: |
+ Thanks for opening this pull request! `Dangerbot` will test out your code and reply in a bit with some pointers and requests.
+ There may be some errors, **but don't worry!** We're here to help! 👍🎉😄
 # Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
 
 # Comment to be posted to on pull requests merged by a first time user
-firstPRMergeComment: >
-  Congrats on merging your first pull request! 🙌🎉⚡️
+firstPRMergeComment: |
+ Congrats on merging your first pull request! 🙌🎉⚡️ 
+ Your code will likely be published to https://code.publiclab.org in the next few days. 
```

Comment:
Hi! Here we may want to keep plain publiclab.org as that's where the code will be published. Would you mind making this change? Thanks!!!!